packages/formation-react/src/components/RightRail/RightRail.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const Sidebar = props => (
+  <div className="medium-screen:vads-u-padding-left--4">{props.children}</div>
+);
+
+Sidebar.propTypes = {
+  classes: PropTypes.string,
+};
+
+export default Sidebar;
```

Comment:
Even though the directory is already renamed (thanks for doing that), can we always update the `Sidebar` var name?

Reply:
ugh, sorry. I did a search and replace and Atom missed these LOL. Should be updated now.